tests/kernel/common/src/errno.c

```diff
@@ -98,4 +101,52 @@ void test_thread_context(void)
 	if (errno != errno_values[N_THREADS]) {
 		rv = TC_FAIL;
 	}
+
+	/* Make sure all the test thread end. */
+	for (int ii = 0; ii < N_THREADS; ii++) {
+		k_thread_join(&threads[ii], K_FOREVER);
+	}
+}
+
+
+#define ERROR_ANY 0xfc
+
+void thread_entry_user(void *p1, void *p2, void *p3)
+{
+#ifdef CONFIG_ARCH_POSIX
+	/* The errno in native posix will be handled by native
+	 * operation system, so we skip it.
+	 */
+	ztest_test_skip();
+#else
+	int got_errno;
+
+	/* assign the error number to C standard errno */
+	errno = ERROR_ANY;
+
+	/* got errno zephyr stored */
+	got_errno = *(z_errno());
+
+	zassert_equal(errno, got_errno, "errno is not corresponding.");
+#endif
+}
+
+/**
+ * @brief Verify errno works well
+ *
+ * @details Check whether assign a C standard errno is successfully
```

Comment:
It better to change it like:

"Check whether a C standard errno can be stored ...."

Reply:
Got it fixed. Thank you for correcting this!